packages/cli/commands/functions/deploy.js

```diff
@@ -72,15 +76,19 @@ exports.handler = async options => {
       })
     );
   } catch (e) {
-    spinner && spinner.stop && spinner.stop();
+    SpinniesManager.fail('loading', {
+      text: i18n(`${i18nKey}.loadingFailed`, {
+        account: uiAccountDescription(accountId),
+        functionPath,
+      }),
+    });
+
     if (isHubSpotHttpError(e) && e.status === 404) {
       logger.error(
         i18n(`${i18nKey}.errors.noPackageJson`, {
           functionPath,
         })
       );
-    } else if (isHubSpotHttpError(e) && e.status === 400) {
-      logger.error(e.error.message);
```

Comment:
I removed this because `logError()` should be enough. I tested and the error still looks good. The error is slightly shorter (not entirely sure why), but it still contains all of the important info.